package.json

```diff
@@ -2,125 +2,24 @@
   "name": "@rilldata/rill",
```

Comment:
Would it make sense to move the `@rilldata/rill` name to `web-frontend/package.json`, since the NPM publish is only that package? This top-level package.json should never be distributed. We could call the root just `@rilldata` or `dev` or something (Prisma calls their root `dev-env`: https://github.com/prisma/prisma/blob/main/package.json)

Reply:
Hmmm makes sense. Would just have to make sure in the CLI the correct package.json is read for version.